utils/record_video.py

```diff
@@ -3,27 +3,71 @@
 
 import gym
 import numpy as np
-from stable_baselines3.common.vec_env import VecVideoRecorder, VecFrameStack, VecNormalize
+from stable_baselines3.common.vec_env import (
```

Comment:
Why did you break that down? The line was not too long, no?

Reply:
It was too long according to flake more than 79 character and this way it looks more clean and easy to read

Reply:
I added the flake8 config here: https://github.com/DLR-RM/rl-baselines3-zoo/blob/master/setup.cfg#L8

`max-line-length = 127`

Reply:
I would prefer breaking lines only when above that limit. Otherwise, you end up with files with tons of lines...

Reply:
How can i set my editor to use this config?

Reply:
I don't know :/ I'm personally using `make lint` to check the code (don't bother for warnings that don't come from your edits)
if you are using pycharm, I'm pretty you change that in the project settings.